conf/messages.en

```diff
@@ -294,6 +294,7 @@ messages__common__address_postcode_hint = For example NE12 8ER
 messages__error__postcode = Enter the postal code
 
 messages__select_the_address__title = Select the address
```

Comment:
can we not use messages__select_the_address__title key instead of creating new one